src/run.ts

```diff
@@ -209,7 +209,7 @@ async function deploy(options: { stage: string }) {
   const stage = options.stage;
   const runner = new LabeledProcessRunner();
   await prepare_services(runner);
-  const deployCmd = ["cdk", "deploy", "--context", `stage=${stage}`, "--all"];
+  const deployCmd = ["cdk", "deploy", "--context", `stage=${stage}`, "--method=direct", "--all"];
```

Comment:
This should make the deploys a bit faster. Instead of using a changeset it directly updates the stack. Apparently it makes it faster, but you lose some progress tracking while it is running. I figured that was better. 🤷